package.json

```diff
@@ -65,4 +65,4 @@
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": true
   }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
done ,updated @krishguptadev

Reply:
Please mark the requested changes as resolved after they are resolved